Fix connector configuration comparison

mattburgess opened the pull request:
The API returns the 'name' of a connector in its response. Add this
attribute to the configuration payload so that a comparison of
dictionaries will work correctly.

Commit:
Fix connector configuration comparison

The API returns the 'name' of a connector in its response. Add this
attribute to the configuration payload so that a comparison of
dictionaries will work correctly.

dan-hill2802 commented on main.py:
```diff
@@ -148,6 +148,7 @@ def configure_confluent_kafka_consumer(event, args):
         "path.format": args.path_format,
         "locale": args.locale,
         "timezone": args.timezone,
+        "name": args.connector_name,
```
I don't like the idea of adding a key to the config that doesn't belong. How about instead removing the "name" key from `current_config`?

Commit:
Reverse the dict adjustments that allow config comparisons to work

Instead of adding an attribute that isn't required, remove the attribute that
is surplus to requirements.
